index.html

```diff
@@ -4375,27 +4726,29 @@ <h3 id="Warrior">Warrior</h3>
                      making hostiles aggressive. </strong>
                   <p>
                      Until the start of your next turn, you become <a class="condition" data-tooltip-container>resistant</a> to all physical
-                     damage. All enemies within 10ft of you become <a class="condition" data-tooltip-container>taunted</a>. 1 WILL point to
+                     damage and one other damage type of your choice. All enemies within 15ft of you become 
```

Comment:
"one other type of your choice" - the decision here slows down combat and is unthematic.  With the power boosts across the board, I see no issues with being resistant with all types of damage.  It's still only for 18s a day and you're not invulnerable.

Reply:
I think that makes it too generically good, remember that legendary resistance halves damage now rather than negating. At level 1 there's a strong incentive for everyone to take this. Also remember that we went through warrior tank abilities to give them some counterplays, otherwise they're just too good